fix usage of chemtex in latex (bug #447559)
usptream fix from Hans Hagen
---
 tex/generic/context/ppchtex.noc |    6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

Index: context-2007.12.18/tex/generic/context/ppchtex.noc
===================================================================
--- context-2007.12.18.orig/tex/generic/context/ppchtex.noc	2007-12-24 09:34:17.000000000 +0100
+++ context-2007.12.18/tex/generic/context/ppchtex.noc	2007-12-24 09:35:20.000000000 +0100
@@ -30,14 +30,10 @@
 
 %D First we load some auxiliary macro's:
 
-\let\normalunexpanded\unexpanded
-
 \input supp-mis.tex  \let\writestatus\undefined
 \input syst-gen.tex
 \input syst-fnt.tex
 
-\let\unexpanded\normalunexpanded
-
 %D after which we can go on with:
 
 \unprotect
@@ -211,4 +207,6 @@
   \setupchemical[\c!resolution=\outputresolution]
 \fi
 
+\let\unexpanded\normalunexpanded
+
 \protect \endinput
